src/eventsAfter.ts

```diff
@@ -28,7 +28,7 @@ export async function eventsAfter (args: string[]): Promise<void> {
     }
     console.log(JSON.stringify(messages, null, 2))
   } catch (error) {
-    exit(error.message)
+    exit((error as Error).message)
```

Comment:
Instead of all of these `(x as Error)` casts, it'd make more sense to modify `exit` to take `unknown`, and do as follows:

1. `if (typeof x === 'string') msg = x; else if (x instanceof Error) msg = x.message; else msg = String(x)`